masterlist.yaml

```diff
@@ -2784,31 +2784,102 @@ plugins:
         name: 'Even Better Mod Descriptions'
     msg:
       - <<: *alsoUseX
-        subs: [ '[Even Better Mod Descriptions GOTY Fixed](https://www.nexusmods.com/fallout4/mods/74459/)' ]
-        condition: 'checksum("EBMD_AllInOne.esp", 3BC4243F) or checksum("EBMD_AllInOne.esp", DC398CCF)'
+        subs: [ '[Even Better Mod Descriptions Fixed](https://www.nexusmods.com/fallout4/mods/74459/)' ]
+        condition: 'checksum("EBMD_AllInOne.esp", 4B1243AE) or checksum("EBMD_AllInOne.esp", 94EE726D)'
+    clean:
+      # GOTY(Main Version 4.1)
+      - crc: 0x4B1243AE
+        util: 'FO4Edit v4.0.4'
+      # GOTY (No Perks 4.1)
+      - crc: 0x94EE726D
+        util: 'FO4Edit v4.0.4'
+      # GOTY(Main Version 4.1) Fixed
+      - crc: 0x14704334
+        util: 'FO4Edit v4.0.4'
+      # GOTY(Main Version 4.1) Fixed UFO4P
+      - crc: 0xF0DDE373
+        util: 'FO4Edit v4.0.4'
+      # GOTY (No Perks 4.1) Fixed
+      - crc: 0xBDD21B9E
+        util: 'FO4Edit v4.0.4'
+      # GOTY (No Perks 4.1) Fixed UFO4P
+      - crc: 0x08B0A737
+        util: 'FO4Edit v4.0.4'
+  - name: 'EvenBetterMDesc.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/36105/'
+        name: 'Even Better Mod Descriptions'
+    msg:
+      - <<: *alsoUseX
+        subs: [ '[Even Better Mod Descriptions Fixed](https://www.nexusmods.com/fallout4/mods/74459/)' ]
+        condition: 'checksum("EvenBetterMDesc.esp", 31185F01)'
```

Comment:
If you add version info to you files, you could replace the `checksum()` with `version()`.
It's far easier to manage than checksums and is easy enough to add version info through xEdit.

```yaml
        condition: 'checksum("EvenBetterMDesc.esp", 31185F01)'
```

Reply:
I can show you on discord if you're interested.